sktime/forecasting/conditional_invertible_neural_network.py

```diff
@@ -1,4 +1,4 @@
-"""Conditional Invertible Neural Network (cINN) for forecasting."""
```

Comment:
I think the docstring name can stay, thats how it is in the paper

Reply:
and what about the class docstring? I also changed that to `CINN`

Reply:
I'd leave that `cINN`, I think that's consistent with the publications. The issue is only with the class name, which is not PEP compliant.